MixItUp.Base/Commands/CommandBase.cs

```diff
@@ -152,22 +157,32 @@ public Task Perform(UserViewModel user, IEnumerable<string> arguments = null)
 
         protected virtual async Task PerformInternal(UserViewModel user, IEnumerable<string> arguments, CancellationToken token)
         {
-            for (int i = 0; i < this.Actions.Count; i++)
+            if (this.IsRandomized)
```

Comment:
Possible simpler way to do this:

- Create a empty list of Actions
- If IsRandomized, randomly add 1 action to the list
- Else, add all the actions
- Then run rest of code normally